microsoft-edge/webview2/release-notes/index.md

```diff
@@ -67,11 +67,344 @@ See also:
+
+* [`COREWEBVIEW2_SAVE_AS_KIND` Enum](/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.2739.15&preserve-view=true#corewebview2_file_save_as_kind)
+   * `COREWEBVIEW2_SAVE_AS_KIND_DEFAULT`
+   * `COREWEBVIEW2_SAVE_AS_KIND_HTML_ONLY`
+   * `COREWEBVIEW2_SAVE_AS_KIND_SINGLE_FILE`
+   * `COREWEBVIEW2_SAVE_AS_KIND_COMPLETE`
+
+* [`COREWEBVIEW2_SAVE_AS_UI_RESULT` Enum](/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.2739.15&preserve-view=true#corewebview2_save_as_ui_result_kind)
+   * `COREWEBVIEW2_SAVE_AS_UI_RESULT_SUCCESS`
+   * `COREWEBVIEW2_SAVE_AS_UI_RESULT_INVALID_PATH`
+   * `COREWEBVIEW2_SAVE_AS_UI_RESULT_FILE_ALREADY_EXISTS`
+   * `COREWEBVIEW2_SAVE_AS_UI_RESULT_KIND_NOT_SUPPORTED`
+   * `COREWEBVIEW2_SAVE_AS_UI_RESULT_CANCELLED`
+   
+---
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+<!-- todo: populate, or note None, or link to other section (expand the latter's heading to specify version #) -->
+
+
+<!-- ---------- -->
+###### Runtime and SDK
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+
+<!-- ---------- -->
+###### SDK-only
+
+
+<!-- end of Aug 2024 Release SDK -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2783-prerelease
+
+Release Date: August 26, 2024
+
+[NuGet package for WebView2 SDK 1.0.2783-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2783-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 129.0.2783.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+No Experimental APIs have been added in this Prerelease SDK.
+
+---
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+No APIs have been promoted from Experimental to Stable in this Prerelease SDK.
+
+---
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+<!-- ---------- -->
+###### Runtime and SDK
+
+* Re-enabled the default behavior of `SetUserAgent`: by default, `SetUserAgent` is effective for cross-origin iframes. 
+<!-- todo: clarify "being effective for" -->
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Enabled the interactive dragging feature by default.  See `edge-webview-interactive-dragging` in [WebView2 browser flags](../concepts/webview-features-flags.md).
+* Disabled `IsolateSandboxedIframes` for WebView2.
+* Fixed an issue where WebView creation fails when multiple instances are launched at the same time.  ([Issue #4731](https://github.com/MicrosoftEdge/WebView2Feedback/issues/4731))
+* Fixed a bug in WinRT JavaScript projection where caching existing properties in Proxy and Function like name<!-- todo: clarify/format, give example api names.  does name literally contain "Proxy" or "Function"?  or a name that's like the name of the proxy or the function? --> caused an error due to name collision.
```

Comment:
```suggestion
* Fixed a bug in WinRT JavaScript projection where caching existing properties in objects whose name contains `Proxy` or `Function` caused an error due to name collision.
```
clarify.  does name literally contain "Proxy" or "Function"?  or a name that's like the name of the proxy or the function?

orig commit's wording:
* Fixed a bug in WinRT JavaScript projection where caching existing properties in Proxy and Function like name caused an error due to name collision.
this Suggestion's wording:
* Fixed a bug in WinRT JavaScript projection where caching existing properties in objects whose name contains `Proxy` or `Function` caused an error due to name collision.
alt wording:
* Fixed a bug in WinRT JavaScript projection where caching existing properties in objects whose name is like the name of the proxy or the function caused an error due to name collision.

@peiche-jessica @nishitha-burman

Reply:
Suggestion sounds good to me, thanks.